### ISSUE TITLE
Locale?

### ISSUE THREAD

**Issue by burtonator** [headzoo/react-moment#74]
It's unclear from the docs but is the local automatically determined by default?

**Comment by weareoutman** [comment]
Yes, I also found that `react-moment` will *force* set locale to `"en"`. After investigating, the problem should be caused by one of its deps [moment-duration-format](https://github.com/jsmreese/moment-duration-format). See https://github.com/jsmreese/moment-duration-format/blob/master/lib/moment-duration-format.js#L1660

**Comment by headzoo** [comment]
Closing old open issue during cleanup.
